zeebe/engine/src/main/java/io/camunda/zeebe/engine/Engine.java

```diff
@@ -46,7 +46,7 @@ public class Engine implements RecordProcessor {
       "Expected to process record '%s' without errors, but exception occurred with message '%s'.";
 
   private static final EnumSet<ValueType> SUPPORTED_VALUETYPES =
-      EnumSet.range(ValueType.JOB, ValueType.COMPENSATION_SUBSCRIPTION);
+      EnumSet.range(ValueType.JOB, ValueType.MESSAGE_CORRELATION);
```

Comment:
You have not forgotten anything  🤣

Reply:
The developer handbook saved me on this 😆